app/src/main/java/org/jellyfin/androidtv/integration/dream/composable/DreamHeader.kt

```diff
@@ -41,11 +44,14 @@ fun DreamHeader(
 			visible = showLogo,
 			enter = fadeIn(),
 			exit = fadeOut(),
-			modifier = Modifier.height(41.dp),
+			modifier = Modifier.height(41.dp).alpha(alpha),
 		) {
 			Image(
 				painter = painterResource(R.drawable.app_logo),
 				contentDescription = stringResource(R.string.app_name),
+				// We also set the alpha for the non-header logo, as this can
+				// be displayed as a fallback
+				modifier = Modifier.alpha(alpha)
```

Comment:
Why set the alpha for both the row and the logo? That just multiplies it. Right now the logo is barely readable.

Reply:
The problem is the row has text elements, such as the name, I want to dim down too so I applied it to the row

Unfortunately, I also wanted to dim the logo too, for the times where we fail to load a backdrop (so the brightness doesn't go up or down), so I unintentionally multiplied these alphas.

I could manually do alpha on each element of the row (excluding the logo), or add a constructor to the Logo to set the alpha to avoid multiplying this into the row? I'm not sure what the best approach would be

Reply:
Just apply it to the row only?